src/status_im/data_store/realm/schemas/base/account.cljs

```diff
@@ -242,3 +242,7 @@
 (def v22 (update (update v21 :properties dissoc :mainnet-warning-shown?)
                  :properties merge
                  {:mainnet-warning-shown-version {:type :string :optional true}}))
+
+(def v23 (assoc-in v22
+                   [:properties :keycard-key-uid]
```

Comment:
could you explain the role of this uid from a security pov? this part of the db is only encrypted at the app level and not the account level

Reply:
@yenda key-uid is uid of master key on keycard so we can link it to status account. 

More info on key UID from keycard docs https://status.im/keycard_api/sdk_create_wallet.html
> The UID is tied to the key itself (is derived from the public key) so it will change if the wallet on card is replaced. The key UID is also part of the response of the applet selection command, so the wallet can be identified immediately upon selection.

Since it's derived from public key i assume it's safe to store it on client unencrypted.